src/components/Auth/api.ts

```diff
@@ -87,22 +87,46 @@ export default class AuthApi extends RestApi {
     return this.fetchUser(json.localId);
   }
 
-  async updateConfig(
+  async updateAllowDuplicateEmails(
     allowDuplicateEmails: boolean
-  ): Promise<{ signIn: { allowDuplicateEmails: boolean } }> {
+  ): Promise<boolean> {
     const { json } = await this.patchRequest(`${this.baseEmulatorUrl}/config`, {
       signIn: { allowDuplicateEmails },
     });
 
-    return json;
+    return json.signIn?.allowDuplicateEmails;
   }
 
-  async getConfig(): Promise<AuthUser> {
-    const { signIn } = await (
-      await fetch(`${this.baseEmulatorUrl}/config`)
-    ).json();
+  async updateUsageMode(
+    usageMode: UsageModes
+  ): Promise<{ signIn: { allowDuplicateEmails: boolean } }> {
```

Comment:
typing here is wrong -- it should return `Promise<UsageModes>`
```suggestion
  ): Promise<UsageModes> {
```

Reply:
removed the `updateUsageMode` function per https://github.com/firebase/firebase-tools-ui/pull/654#discussion_r732312481